playbooks/eth1_ip.yml

```diff
@@ -5,7 +5,7 @@
     - name: Install nmstate package
       package:
         name: nmstate
-        state: present
+        state: latest
```

Comment:
This will install the latest version that is available in the configured repositories, but this might be an older version than 0.4. The problem with using "latest" is that this will make the playbook slower if the package is already installed. Therefore it seems that "present" is a better solution.

Reply:
On my Centos8 virtual machine, it was painful to install nmstate 0.4. The steps are: 
- `sudo dnf copr enable nmstate/nispor`
- `sudo dnf install nispor`
- `sudo dnf copr enable nmstate/NetworkManager`
- `sudo dnf install NetworkManager`
- `sudo dnf copr enable nmstate/nmstate`
- `sudo dnf install nmstate`


Originally I can not install nmstate 0.4 with Ansible because of lacking dependencies. Then I installed nmstate 0.4 mannually by following above steps, and I removed/uninstalled it. After removing/uninstalling it, I can install nmstate 0.4 with Ansible. On CI, maybe dependencies already satisfied, then we can install nmstate 0.4 with Ansible?

Reply:
I was more worried about users getting a clear error that they need nmstate 0.4 or later. Btw. you should not need to do all the install steps. After the three dnf copr enable steps, `dnf install nmstate` should pull in NetworkManager and nispor in the right versions. This will be better with the next CentOS release. It might be better to test with Fedora for now.

Reply:
Sure, I will download a fedora image to test on it.

Reply:
@tyll I tested on Fedora 32 Virtual machine, I can only install nmstate 0.3.3 via Ansible. Then I mannually installed nmstate 0.4 the same way as I did on CentOS8 VM. Maybe we can install nmstate 0.4 via Ansible command module:
```
command:
- sudo dnf copr enable nmstate/nispor
- sudo dnf copr enable nmstate/NetworkManager
- sudo dnf copr enable nmstate/nmstate
- sudo dnf install nmstate

Reply:
how do you like that, @tyll

Reply:
In my opinion it should be fine :-)

Reply:
I am not convinced that this is the best approach. It seems odd to have the complex copr repos as part of the examples. I would rather keep it as it was before and mentione the requirement for 0.4 or later in the README. Enabling the copr repos could be another example playbook.

What about Fedora 33 that was just released? Does it contain Nmstate 0.4?

Reply:
Sure, and I will also test on Fedora 33 later on. The Centos 8 and Fedora 32 VDI (VirtualBox image) was download from https://www.osboxes.org/fedora/#fedora-32-info. I may also gather or config #fedora-33-info for testing purpose.

Reply:
@tyll ,Should I create a new example playbook about enabling the copr repos and installing nmstate 0.4?

Reply:
yes, sounds like a good idea.

@ffmancera is it intentional that Fedora 33 does not ship nmstate 0.4?

Reply:
we can not run the cmd  `sudo dnf copr enable nmstate/nispor`, `- sudo dnf copr enable nmstate/NetworkManager`, `- sudo dnf copr enable nmstate/nmstate`   on fedora 33, we got following error: 

```
Error: This repository does not have any builds yet so you cannot enable it now.
```

Reply:
open a new PR https://github.com/nmstate/ansible-nmstate/pull/25

Reply:
> we can not run the cmd `sudo dnf copr enable nmstate/nispor`, `- sudo dnf copr enable nmstate/NetworkManager`, `- sudo dnf copr enable nmstate/nmstate` on fedora 33, we got following error:
> 
> ```
> [liveuser@localhost-live ~]$ sudo dnf copr enable nmstate/nmstate -y
> Enabling a Copr repository. Please note that this repository is not part
> of the main distribution, and quality may vary.
> 
> The Fedora Project does not exercise any power over the contents of
> this repository beyond the rules outlined in the Copr FAQ at
> <https://docs.pagure.org/copr.copr/user_documentation.html#what-i-can-build-in-copr>,
> and packages are not held to any quality or security level.
> 
> Please do not file bug reports about these packages in Fedora
> Bugzilla. In case of problems, contact the owner of this repository.
> Error: This repository does not have any builds yet so you cannot enable it now.
> ```

@ffmancera please take a look

Reply:
This error is because nispor and NetworkManager repositories are not being built for Fedora 33. I've fixed it, it should work now. Could you check it out? @liangwen12year

Reply:
@ffmancera I tested, I still got the same error

Reply:
@liangwen12year Yes, the Copr repository is still building. With need to give them more time. I will let you know to retry again when ready.

Reply:
@ffmancera got it, thanks for notifying me